lib/Optimizer/GraphOptimizer/GraphOptimizer.cpp

```diff
@@ -2922,6 +2907,29 @@ bool OptimizeQuantization::run(Function *F, const CompilationContext &cctx) {
   return changed;
 }
 
+void glow::convertQuantizedConstants(Function *F, CompilationContext &cctx) {
+  for (auto &node : F->getNodes()) {
+    if (auto *Q = dyn_cast<QuantizeNode>(&node)) {
```

Comment:
nit: could early continue to reduce amount of indentation, e.g.

```cpp
auto *Q = dyn_cast<QuantizeNode>(&node);
if (!Q) {
  continue;
}
```

Reply:
Done